views/404.ejs

```diff
@@ -0,0 +1,8 @@
+<%- include('includes/head.ejs') %>
+</head>
+
+<body>
+    <%- include('includes/navigation.ejs') %>
+    <h1>Page Not Found!</h1>
```

Comment:
here we can show status code and some cool images like the one GitHub show when u tries to access something which is unauthorized or doesn't exist.

Reply:
Will update it . give me some nice 404 error imgae of gif file 😂

Reply:
Added 404 Error image as background